### ISSUE TITLE
Home Assistant tile is black

### ISSUE THREAD

**Issue by anonymousaga** [AlexxIT/WebRTC#444]
When I launch the dashboard tab containing the WebRTC MJPEG tile, the image shows correctly, like this:
<img width="513" alt="Screenshot 2023-03-01 at 21 14 16" src="https://user-images.githubusercontent.com/68388009/222313336-3f8d84b2-0515-4a9e-aef7-8269aef6e393.png">

However, if I change to a different tab and then come back to the tab with the WebRTC MJPEG tile, then it appears black and I have to refresh the browser to see the cameras. 
<img width="495" alt="Screenshot 2023-03-01 at 21 14 23" src="https://user-images.githubusercontent.com/68388009/222313456-0d578f24-35a0-4de3-91b6-dc529e0af593.png">

Once again, this only happens with MJPEG cameras, but my RTSP camera doesn't have this issue.

This is only a minor inconvenience, but I wanted to bring it up. Thanks in advance.

**Comment by anonymousaga** [comment]
```
type: custom:webrtc-camera
url: http://192.168.0.131/webcam/?action=stream/
ui: true
title: Top
muted: true
```
Here is my config for the card

**Comment by AlexxIT** [comment]
Try to setup stream in the go2rtc config

**Comment by AlexxIT** [comment]
Yes. But you can add stream to go2rtc config and use stream by name in the WebRTC card. Maybe this will help with your problem
